04-store-api/starter/populate.js

```diff
@@ -0,0 +1,21 @@
+require("dotenv").config()
+
+const connectDB = require("./db/connect")
+const Product = require("./models/product")
+
+const jsonProducts = require("./products.json")
+
+const start = async () => {
+  try {
+    await connectDB(process.env.MONGO_URI)
+    await Product.deleteMany()
```

Comment:
Does the name `populate.js` communicate that this script takes a destructive action? Can we think of a different name that might save another developer from accidentally wiping their db?